### PR TITLE
Venue Reaction Toggle

### PR DESCRIPTION
**Pull request by jacoblurie29**
One endpoint to rule them all.

(ethan has never seen the lord of the rings...)

image
--
image
--
Screen Shot 2023-04-17 at 10 23 39 PM

### CODE REVIEW

File: src/queue/setup/workers.setup.ts [hunk 1]
```diff
@@ -25,6 +25,8 @@ const workerOptions: WorkerOptions = {
  * @returns {Promise<void>} completion of the worker after handling the job
  */
 const workerHandler = async (job: Job<NightlightQueueJob>) => {
+  console.log(`Worker # ${job.id} added. Job type: ${job.data.type}.`);
```
Comment (ethanratnofsky) [inline src/queue/setup/workers.setup.ts:28]: Add "[Queue]" prefix

File: src/queue/workers.ts [hunk 2]
```diff
@@ -65,6 +65,7 @@ export const expireReaction = async (
   emoji: string
 ) => {
   try {
+    console.log('EXPIRING REACTION: ' + userId + ' ' + venueId + ' ' + emoji);
```
Comment (ethanratnofsky) [inline src/queue/workers.ts:68]: Add "[Queue]" prefix

File: src/routes/venues.router.ts [hunk 3]
```diff
@@ -16,8 +15,7 @@ venuesRouter.post('/', createVenue);
 venuesRouter.get('/:venueId', getVenue);
 venuesRouter.get('/', getVenues);
 venuesRouter.delete('/:venueId', deleteVenue);
-venuesRouter.post('/:venueId/reaction', addReactionToVenue);
-venuesRouter.delete('/:venueId/reaction', deleteReactionFromVenue);
 venuesRouter.patch('/:venueId', updateVenue);
+venuesRouter.patch('/:venueId/reaction', toggleReactionToVenue);
```
Comment (ethanratnofsky) [inline src/routes/venues.router.ts:19]:
```suggestion
venuesRouter.patch('/:venueId/react', toggleReactionToVenue);
```

**Review by ethanratnofsky — CHANGES_REQUESTED** [review]

**Comment by zineanteoh** [comment]
I have also not seen lord of the rings

**Review by ethanratnofsky — APPROVED** [review]

**Pull request merged**